interfaces/builtin/steam_support.go

```diff
@@ -399,7 +400,7 @@ func init() {
 	registerIface(&steamSupportInterface{commonInterface{
 		name:                  "steam-support",
 		summary:               steamSupportSummary,
-		implicitOnCore:        false,
+		implicitOnCore:        release.OnCoreDesktop,
```

Comment:
do we want to add a test about this?

Reply:
yes, maybe something similar to `TestStaticInfo` in other interfaces with a variant for core desktop?